scripts/linters/pylint_extensions.py

```diff
@@ -1062,30 +1062,31 @@ def process_module(self, node):
             node: astroid.scoped_nodes.Function. Node to access module content.
         """
 
-        in_multi_line_comment = False
-        multi_line_indicator = b'"""'
+        is_class_or_function = False
```

Comment:
don't start a variable name with verb. [Dito below]

Maybe in_class_or_function or line_is_calls_or_function (I don't have a proper context of this variable)?

Reply:
Done.